src/components/DropdownButton/DropdownButton.tsx

```diff
@@ -5,7 +5,7 @@ import styles from "./DropdownButton.module.css";
 
 type Props = {
   className?: string;
-  children?: ReactNode;
+  text?: ReactNode;
```

Comment:
> I'm also wondering if we want this API change at all since Button uses children. It feels a little out-of-sync

hmmmm good question. could we use this heuristic?
- If it can be typed as a `string`, use `text` (we're always expecting simple input, so there's benefit to making the API match that expectation)
- If it has to be typed as `ReactNode`, use `children` (we're allowing potentially more complicated input, so "text" may not be an accurate prop name)

Reply:
I think `DropdownButton` is mostly a utility component (only used in implementing our other components), so the API difference with Button isn't confusing.

Are we going to update the external-facing `Dropdown.Button` and `Dropdown.Label` API as well though? If not, less sure the benefit of this change

Reply:
>Are we going to update the external-facing Dropdown.Button and Dropdown.Label API as well though? If not, less sure the benefit of this change. I think those 3 should stay in sync (in whichever direction we choose)

That was actually where it started feeling really strange to me – `Dropdown.Button` has to be able to accept other types because it can be passed a function: https://github.com/chanzuckerberg/edu-design-system/blob/d8e6f690611a8b8bdccdaa5b8fe7bcbc7cee4ed6/src/components/Dropdown/Dropdown.stories.tsx#L109

Reply:
oo shoot, right 😓  Feels easier to me to leave `children` then

Reply:
Cool, I'll close this and the ticket then. Thanks!